plugin-guide.md

```diff
@@ -204,3 +204,88 @@ For example, if external keys are connected to Pinery, then a filter might want
 to filter on runs. A filter could query Pinery and get all the external
 identifiers associated with that run and then construct a query based on those
 to match workflow runs that use any of those identifiers.
+
+# Provided Implementations
+This core implementation provides several plugins independent of external
+systems.
+
+## Consumable Resources
+Consumable resources provided in Víðarr core.
+
+### Manual Override
+Allows overriding a consumable resource to permit workflows to run even if they
+would hit a limit.
+
+To set one up, a cache file is needed to store the workflow run IDs that have
+been allowed. It must be in a directory writable by the server. The `"inner"`
+property is the configuration for another consumable resource to wrap.
```

Comment:
1. Why does this need to be on disk?
2. 'the inner property is the configuration for another consumable resource to wrap' is confusing. Does that mean the entire config for, ex max-in-flight gets duplicated inside 'inner'?

Reply:
1. Where should it go? I assume it needs to be persistent.
2. Yes! Though not necessarily duplicated. You'd want to have one max-in-flight inside the manual-override wrapper. If you had two max-in-flights, the top-level one could not be overridden.

Reply:
1. I would think it would go in the database if it needed to persist, or else just memory
2. Makes sense, got confused with shesmu where every thing is a different config file. I still think the wording is confusing though - the inner property is the consumable resource that the manual override wraps, the current wording sounds like a third consumable resource is coming in and getting wrapped _by_ 'inner' somehow

Reply:
1. In memory it is.
2. I've rewritten this section with an example. Let me know if that makes more sense.